osf/migrations/0007_auto_20170404_1415.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2017-04-04 19:15
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0006_add_jsonb_index_for_fileversions'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='institution',
+            name='delegation_protocol',
+            field=models.CharField(blank=True, choices=[(b'cas-pac4j', b'CAS by pac4j'), (b'oauth-pac4j', b'OAuth by pac4j'), (b'saml-shib', b'SAML by shibboleth'), (b'', b'No Delegation Protocol')], default=b'', max_length=15),
```

Comment:
choices should follow this implementation https://github.com/CenterForOpenScience/osf.io/blob/084bc28dbf56d614451659655c093ff745850869/osf/models/sanctions.py#L34